wordpress-mobile/WordPress-Android
# 🛑 [DO NOT MERGE] DEBUG: Remove links handling code

> **Warning**: DO NOT MERGE!!!

**PR to test if FB app still opens links on some pages with the WP app.**

To test:
1. Remove old WP app and install it from this PR
2. Open FB app, go to the WPTavern page in Facebook
3. Click a post that links to an article on the wptavern website
4. **Verify** WordPress app opens on My Site

## Regression Notes
1. Potential unintended areas of impact
  N/a

5. What I did to test those areas of impact (or what existing automated tests I relied on)
  N/a

6. What automated tests I added (or what prevented me from doing so)
  N/a

PR submission checklist:
